lib/internal/errors.js

```diff
@@ -356,10 +369,15 @@ function E(sym, val) {
   messages.set(sym, typeof val === 'function' ? val : String(val));
 }
 
-// This creates an error compatible with errors produced in UVException
-// using the context collected in CollectUVExceptionInfo
-// The goal is to migrate them to ERR_* errors later when
-// compatibility is not a concern
+/**
+ * This creates an error compatible with errors produced in the C++
+ * function UVException using a context object with data assembled in C++.
+ * The goal is to migrate them to ERR_* errors later when compatibility is
+ * not a concern.
+ *
+ * @param {Object} ctx
```

Comment:
I went with the JSDoc-style annotations here to make it clearer about what these helpers take and returns